app/android/app/build.gradle

```diff
@@ -75,7 +75,7 @@ afterEvaluate {
 /**
  * Run Proguard to shrink the Java bytecode in release builds.
  */
-def enableProguardInReleaseBuilds = false
+def enableProguardInReleaseBuilds = true
```

Comment:
Able to confirm Person credential flow still works with this turned on?

Reply:
@bryce-mcmath Proguard does optimization and shrinking for a bundle in Android so I don't think it will impact app functionality. It should be on where possible IMHO.